src/app/admin/get-started/get-started.component.html

```diff
@@ -115,9 +117,10 @@
                   class="form-check-input"
                   type="checkbox"
                   value=""
-                  id="flexCheckDefault"
+                  id="flexCheckDefault3"
```

Comment:
@suvarnakale can you set relavent name for this check box like- ownershipCheckbox like this. Check for all card checboxes